README.md

```diff
@@ -1,3 +1,4 @@
+🤯
```

Comment:
Does it support IE 11???

Also, can we replace it with lodash function??

Reply:
We need to add jQuery before.

Reply:
Could you add jquery in other PR I will rebase my branch on it

Reply:
Ok. If we are going to add jquery, please add performance tests. AFAIK it could affect performance a bit.

https://github.com/GoogleChrome/lighthouse-ci
should be fine

cc @bodia-uz

Reply:
Add e2e tests after

Reply:
Fixed issue with ie

Reply:
Thanks LGTM

Reply:
Will add e2e in other repo.